### DISCUSSION TITLE
How to integrate bucket4j with redis using micronaut

### DISCUSSION THREAD

**Discussion by raghu298** [vladimir-bukhtoyarov/bucket4j#385] [Q&A]
Can anyone suggest how to implement?

**Comment by raghu298** [comment]
@vladimir-bukhtoyarov could you please guide how to integrate redis with bucket4j .
I am newbie to bucket4j

**Reply by raghu298** [reply]
@vladimir-bukhtoyarov i am trying with micronaut framework

**Reply by vladimir-bukhtoyarov** [reply]
I have no ideas. Bucket4j is a library, there are no differences between usage with and without frameworks.
